daemons/execd/remoted_tls.c

```diff
@@ -273,39 +273,39 @@ bind_and_listen(struct addrinfo *addr)
 
     fd = socket(addr->ai_family, addr->ai_socktype, addr->ai_protocol);
     if (fd < 0) {
-        crm_perror(LOG_ERR, "Listener socket creation failed");
-        return -1;
+        crm_err("Listener socket creation failed: %", pcmk_rc_str(errno));
+        return -errno;
```

Comment:
All these returns will have the same problem ... need to save errno immediately after the system call

Reply:
Fixed.